### PR TITLE
dotCMS/core#19868  Containers pagination needs double click to change on templates portlet

### PR DESCRIPTION
**Pull request by hmoreras**
When you have multiple containers and need to paginate to add into a container using the new templates portlet, you need double click in the number to reload the list. We need do this just with one click

### CODE REVIEW

File: src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts [hunk 1]
```diff
@@ -168,8 +166,9 @@ describe('ContainerSelectorComponent', () => {
         const paginatorService: PaginatorService = de.injector.get(PaginatorService);
         spyOn(paginatorService, 'getWithOffset').and.returnValue(observableOf(containers));
         comp.handleFilterChange('');
-
-        expect(comp.currentContainers[0].identifier).toEqual('427c47a4-c380-439f');
-        expect(comp.currentContainers[1].identifier).toEqual('container/path');
+        comp.currentContainers.subscribe((items: DotContainer[]) => {
+            expect(items[0].identifier).toEqual('427c47a4-c380-439f');
+            expect(items[1].identifier).toEqual('container/path');
```
Comment (fmontes) [inline src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts:171]: I think you should do this againts the HTML attr.

File: src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.ts [hunk 2]
```diff
@@ -164,7 +164,7 @@ export class SearchableDropdownComponent
     }
 
     ngAfterContentInit() {
-        this.totalRecords = this.totalRecords || this.data.length;
+        this.totalRecords = this.totalRecords || this.data?.length;
```
Comment (fmontes) [inline src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.ts:167]: We need this?

**Review by fmontes — CHANGES_REQUESTED** [review]

File: src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.ts (hunk 2 above)
Comment (hmoreras) [reply src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.ts:167]: in this component yes.

File: src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts (hunk 1 above)
Comment (hmoreras) [reply src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts]: done

**Review by fmontes — APPROVED** [review]

**Review by ivorpad — APPROVED** [review]

**Review by alfredo-dotcms — APPROVED** [review]

**Pull request merged**